src/fhirtypes/ElementDefinition.ts

```diff
@@ -2058,6 +2136,47 @@ export class ElementDefinition {
     return slice;
   }
 
+  copyFromParent(profiledElement: ElementDefinition, structDef: StructureDefinition): void {
```

Comment:
Am I missing something major, or is `copyFromParent` only used in `copyFromParent`?